public_documents.html

```diff
@@ -89,16 +89,16 @@ <h1>
                     <td>
                         <ul>
                             <li>
-                                <a href="https://www.omaspecworks.org/managing_non-ip_devices_in_cellular_iot_networks/">OMA Lightweight M2M 1.1:Managing Non-IP Devices in Cellular IoT Networks</a>
+                                <a href="https://www.openmobilealliance.org/documents/whitepapers/LwM2M-Managing%20Non-IP%20Devices%20in%20Cellular%20IoT%20Networks.pdf">OMA-LwM2M_1.1_:Managing-Non-IP-Devices-in-Cellular-IoT-Networks.pdf</a>
                             </li>
                             <li>
-                                <a href="https://www.omaspecworks.org/whitepaper-12-5-2018/">OMA Lightweight M2M 5G, IoT, UN-SDG</a>
+                                <a href="https://www.openmobilealliance.org/documents/whitepapers/LwM2M-5G,%20IoT,%20UN-SDG.pdf">OMA-LwM2M-5G_IoT_UN-SDG-20180512.pdf</a>
                             </li>
                             <li>
-                                <a href="https://www.openmobilealliance.org/documents/whitepapers/OMA-WP-State-of-the-LwM2M-Standard-20200114-C.pdf">OMA Lightweight M2M - Ready For 5G, OMA-WP-State-of-the-LwM2M-Standard-20200114-C.pdf</a>
+                                <a href="https://www.openmobilealliance.org/documents/whitepapers/OMA-WP-State-of-the-LwM2M-Standard-20200114-C.pdf">OMA-LwM2M-Ready-For-5G:OMA-WP-State-of-the-LwM2M-Standard-20200114-C</a>

```

Comment:
These are two documents. The OMA LwM2M Ready for 5G needs to be downloaded from [here](https://omaspecworks.org/wp-content/uploads/2020/01/Whitepaper-11.12.19.pdf) and uploaded it to the TFP whitepapers.

Reply:
Actually, this is one file with a long name, please see https://www.openmobilealliance.org/documents/whitepapers/OMA-WP-State-of-the-LwM2M-Standard-20200114-C.pdf.
Full name of the whitepaper is 
OMA Lightweight M2M (LwM2M) - Ready For 5G  Candidate - 14 Jan 2020  
Open Mobile Alliance  OMA-WP-State-of-the-LwM2M-Standard-20200114-C .